docs/design/integration_nova.adoc

```diff
@@ -20,16 +20,194 @@ We would want to integrate with Nova to support the same set of user operations,
 
 == Review of OpenStack Workflow
 
+nova call neutron apis
+|===
+|Name |method |params |fields |neutron api |alcor status
+
+|show_network
+|get
+|net_id
+|provider:physical_network,provider:network_type
+|/v2.0/networks/{net_id}
+|/v4/{project_id}/vpcs/{vpcid}
```

Comment:
@Gzure Actually, this API of VPC manager is internal, only accessible by API gateway or other microservices, but not exposed to Nova or any other clients. Instead, the API gateway is the place where external clients could reach Alcor.

For example, we do have a similar API "/project/{projectId}/network/{vpcId}", and we could/should make changes there to make all APIs consistent with Nova so that we don't need to make code change to client, ideally.

https://github.com/futurewei-cloud/alcor/blob/05864239666e273c74b67dc18a72c9ee317d97c6/services/api_gateway/src/main/java/com/futurewei/alcor/apigateway/vpc/VpcWebConfiguration.java#L39